ios/StripeTerminalReactNative.swift

```diff
@@ -352,9 +352,9 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
         let paymentMethodOptions = params["paymentMethodOptions"] as? [AnyHashable : Any] ?? [:]
         let extendedAuth = paymentMethodOptions["requestExtendedAuthorization"] as? Bool ?? false
         let incrementalAuth = paymentMethodOptions["requestIncrementalAuthorizationSupport"] as? Bool ?? false
+        let captureMethod = params["captureMethod"] as? String
 
-
-        let paymentIntentParams = PaymentIntentParameters(amount: UInt(truncating: amount), currency: currency, paymentMethodTypes: paymentMethodTypes)
+        let paymentIntentParams = PaymentIntentParameters(amount: UInt(truncating: amount), currency: currency, paymentMethodTypes: paymentMethodTypes, captureMethod: captureMethod == "automatic" ? .automatic : .manual)
```

Comment:
Is there any Swift way of doing this 
```
captureMethod: captureMethod == "automatic" ? .automatic : .manual
```
check?

Reply:
I was just wondering if there is a way to automatically convert `"automatic" -> .automatic` and `"manual" -> .manual`, and maybe have a default that will throw if it's neither. If this is reasonable then I'm happy :)

Reply:
I see. I'm not aware of any automatic way to map from a string to an enum value. We have this pattern elsewhere, and it looks good to me. Thanks for checking!